dolphinscheduler-dist/src/main/assembly/dolphinscheduler-bin.xml

```diff
@@ -67,16 +67,28 @@
         <fileSet>
             <directory>${basedir}/../dolphinscheduler-tools/target/tools</directory>
             <outputDirectory>tools</outputDirectory>
+            <excludes>
+                <exclude>libs/</exclude>
+                <exclude>bin/</exclude>
+                <exclude>dist-bin/</exclude>
+            </excludes>
         </fileSet>
-
         <fileSet>
-            <directory>${basedir}/../dolphinscheduler-dist/target/dolphinscheduler-dist-${project.version}</directory>
-            <outputDirectory>.</outputDirectory>
+            <directory>${basedir}/../dolphinscheduler-tools/target/tools/libs</directory>
+            <outputDirectory>tools/libs</outputDirectory>
+            <includes>
+                <include>dolphinscheduler-*.jar</include>
+                <include>spring-*.jar</include>
+            </includes>
+        </fileSet>
+        <fileSet>
+            <directory>${basedir}/../dolphinscheduler-tools/target/tools/dist-bin</directory>
+            <outputDirectory>tools/bin</outputDirectory>
```

Comment:
In binary package module `tools`, we only need some spring jar and use dir `dist-bin` to load libs from others server